cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -105,6 +106,23 @@
            "(defelem foo [x]\n  [:foo x])")))
 
   (testing "comment before ending bracket"
+    (is (= (->> ["(defn foo"
+                 "  []"
+                 "  (let [x 1]"
+                 "    ;; test1"
+                 "    )"
+                 "  ;; test2"
+                 "  )"]
+                (str/join "\n")
+                reformat-string
+                str/split-lines)
+           ["(defn foo"
+            "  []"
+            "  (let [x 1]"
+            "    ;; test1"
+            "    )"
+            "  ;; test2"
+            "  )"]))
```

Comment:
I've deviated from the existing format of tests because I really struggle to see what's indented etc. when everything's on a single line with newlines dotted around.

I can pack things up in the same style if you prefer.